storage_service/locations/models/package.py

```diff
@@ -341,6 +341,21 @@ def move_to_location(self, destination_location_uuid):
             self.save()
             self.current_location.space.update_package_status(self)
 
+            # Update pointer file's location information
+            if self.pointer_file_path and self.package_type in (Package.AIP, Package.AIC):
+                root = etree.parse(self.full_pointer_file_path)
+                element = root.find('.//mets:file', namespaces=utils.NSMAP)
+                flocat = element.find('mets:FLocat', namespaces=utils.NSMAP)
+                if self.uuid in element.get('ID', '') and flocat is not None:
+                    flocat.set('{{{ns}}}href'.format(ns=utils.NSMAP['xlink']), self.full_path)
```

Comment:
Tip: `utils.PREFIX_NS` has all the namespaces wrapped in `{}` to make this sort of thing easier.  So this could be `flocat.set('{ns}href'.format(ns=utils.PREFIX_NS['xlink']), self.full_path)` or `flocat.set(utils.PREFIX_NS['xlink'] + 'href', self.full_path)`